precheck/lib/precheck/rules/unreachable_urls_rule.rb

```diff
@@ -29,13 +29,13 @@ def rule_block
         begin
           uri = Addressable::URI.parse(url)
           uri.fragment = nil
-          request = Faraday.new(URI.encode(uri.to_s)) do |connection|
+          request = Faraday.new(URI.encode(uri.normalize.to_s)) do |connection|
```

Comment:
You need to get rid of the `URI.encode` here. This is essentially running a URI encode twice (since that is what is done inside of `normalize` plus some more). This is what is causing that change in that test below 🙃